sql/cast-functions-and-operators.md

```diff
@@ -0,0 +1,13 @@
+# Cast 函数和操作符
+
+## Cast 函数和操作符
```

Comment:
This is the name of the table,
which label should I use here? @andelf

Reply:
Feel free not to use any label here.